src/main/java/com/csc4003/apis/models/Attendee.java

```diff
@@ -42,28 +36,49 @@ public void setAttendeeId(int attendeeId) {
         this.attendeeId = attendeeId;
     }
 
-    public int getBookingId() {
-        return bookingId;
+    public Booking getBooking() {
+        return booking;
     }
 
-    public void setBookingId(int bookingId) {
-        this.bookingId = bookingId;
+    public void setBooking(Booking booking) {
+        this.booking = booking;
     }
 
-    public int getEmployeeId() {
-        return employeeId;
+    public Employee getEmployee() {
+        return employee;
     }
 
-    public void setEmployeeId(int employeeId) {
-        this.employeeId = employeeId;
+    public void setEmployee(Employee employee) {
+        this.employee = employee;
     }
 
     @Override
     public String toString() {
         return "Attendee{" +
                 "attendeeId=" + attendeeId +
-                ", bookingId=" + bookingId +
-                ", employeeId=" + employeeId +
+                ", bookingId=" + booking.getBookingId() +
+                ", employeeId=" + employee.getEmployeeId() +
                 '}';
     }
+
+    // ---- Employee Details Methods
```

Comment:
You may be right on this, my thought was that since attendee is linked to both employee and booking classes and not the other way round, if the user wishes to see the attendees for a booking it'll query the attendee model. If you look at AttendeeService and Attendee Repository you will see that there returns a list of attendee object array with the attendees for that booking. 

I may remove those methods you mentioned because it may be bloated code though, you can just get their details by calling .getEmployee from Attendee class anyway.